guides/paper-iron-layout.md

```diff
@@ -0,0 +1,324 @@
+      <div>Header</div>
+    </paper-toolbar>
+    <div>Content</div>
+  </paper-header-panel>
+</body>
+...
+```
+
+[Demonstration](assets/tall-header.html)
+
+### Adding tabs
+
+Use `paper-tabs` to add tabs to your header:
+
+```hmtl
+...
+<head>
+...
+  <link rel="import" 
+        href="/bower_components/paper-header-panel/paper-header-panel.html">
+  <link rel="import" 
+        href="/bower_components/paper-toolbar/paper-toolbar.html">
+  <link rel="import" 
+        href="/bower_components/paper-icon-button/paper-icon-button.html">
+  <link rel="import"
+        href="/bower_components/paper-tabs/paper-tabs.html">
+  <link rel="import" 
+        href="/bower_components/iron-icons/iron-icons.html">
+  <link rel="import" 
+        href="/bower_components/iron-flex-layout/iron-flex-layout.html">
+...
+<body class="fullbleed vertical layout">
+  <paper-header-panel class="flex">
+    <paper-toolbar class="medium-tall">
+      <paper-icon-button id="navicon"
+                         icon="menu"></paper-icon-button>
+      <!-- flex class forces span to fill space between icons -->
+      <span class="flex">Title</span>
+      <!-- icon displays at right because of span class above -->
+      <paper-icon-button id="morebutton"
+                         icon="more-vert"></paper-icon-button>
+      <paper-tabs class="bottom fit" selected="0">
+        <paper-tab>ONE</paper-tab>
+        <paper-tab>TWO</paper-tab>
+      </paper-tabs>
+    </paper-toolbar>
+    <div>Content</div>
+  </paper-header-panel>
+</body>
+...
+```
+
+[Demonstration](assets/tabs.html)
+
+### Modifying header display and behavior
+
+Use the `mode` attribute of `paper-header-panel` to control how the 
+header displays and responds to scrolling. The list below describes 
+the different valid values for `mode`. See the link below for a 
+demonstration of all modes.
+
+* `standard`: The header appears at a higher level than the content area, 
+  with a drop shadow. Content scrolls under the header.
+* `seamed`: The header appears at the same level as the content area, 
+  with a seam between the two (no drop shadow). Content scrolls under the header.
+* `waterfall`: The header initially presents as seamed. When content scrolls 
+  under the header, the header raises up and casts a drop shadow (as in 
+  standard mode).
+* `waterfall-tall`: Like waterfall, except that the toolbar starts off 
+  tall (3x standard height) and condenses to a standard-height 
+  toolbar as the user scrolls. In this mode, `paper-header-panel` controls
+  the height of the toolbar, so you should not set it yourself (via
+  `medium-tall` or `tall`).
+* `scroll`: The header is seamed with the content and scrolls with the content.
+* `cover`: The content scrolls over the header. This mode is designed to 
+  be used with narrow content (for example cards).
+
+[Demonstration](/elements/paper-header-panel?view=demo:demo/index.html)
+
+## Creating a responsive side navigation
```

Comment:
"a responsive side navigation" seems a little awkward. Should this be:

Creating responsive side navigation

Or:

Creating a responsive navigation drawer